arm-keyvault/2015-06-01/swagger/keyvault.json

```diff
@@ -0,0 +1,463 @@
+            "format": "int32",
+            "description": "Maximum number of results to return."
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "enum": [ "2015-11-01" ],
+            "description": "Azure Resource Manager Api Version."
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/VaultListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Sku": {
+      "properties": {
+        "family": {
+          "type": "string",
+          "description": "SKU family name"
+        },
+        "name": {
+          "type": "string",
+          "description": "SKU name"
+        }
+      },
+      "required": [
+        "family",
+        "name"
+      ]
+    },
+    "AccessPolicyEntry": {
+      "properties": {
+        "tenantId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "Tenant ID of the principal"
+        },
+        "objectId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "Object ID of the principal"
+        },
+        "applicationId": {
+          "type": "string",
+          "format": "uuid",
+          "description": " Application ID of the client making request on behalf of a principal"
+        },
+        "permissions": {
+          "$ref": "#/definitions/Permissions",
+          "description": "Permissions that the principal has for this vault"
+        }
+      }
+    },
+    "Permissions": {
+      "properties": {
+        "keys": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Permissions to keys"
+        },
+        "secrets": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Permissions to secrets"
+        }
+      }
+    },
+    "VaultProperties": {
+      "properties": {
+        "vaultUri": {
+          "type": "string",
+          "description": "URL of the vault"
+        },
+        "tenantId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "Tenant ID"
+        },
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "SKU details"
+        },
+        "accessPolicies": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AccessPolicyEntry"
+          },
+          "description": "Access policies for one or more principals"
+        },
+        "enabledForDeployment": {
+          "type": "boolean",
+          "description": "Enabled or disabled for deployment"
```

Comment:
The following will give more info to the customer
> "Specifies whether Azure Virtual Machines are permitted to retrieve certificates stored as secrets from the key vault."
Got this from the [Key Vault MSDN REST API doc](https://msdn.microsoft.com/en-us/library/mt620025.aspx).
Please update the other descriptions as well.